internal/exportimport/scheduler.go

```diff
@@ -116,3 +106,27 @@ func (s *Server) handleSchedule(ctx context.Context) http.HandlerFunc {
 		w.Write([]byte(http.StatusText(status)))
 	}
 }
+
+func syncFilesFromIndex(ctx context.Context, db *exportimportdb.ExportImportDB, config *model.ExportImport, index string) (int, error) {
+	zipNames := strings.Split(index, "\n")
+	currentFiles := make([]string, 0, len(zipNames))
+	for _, zipFile := range zipNames {
+		if len(strings.TrimSpace(zipFile)) == 0 {
+			// drop blank lines.
+			continue
+		}
+		proposedPath := fmt.Sprintf("%s%s", config.ExportRoot, strings.TrimSpace(zipFile))
```

Comment:
path.Join?

Reply:
`filepath.Join` in case someone runs this on Windows ever

Reply:
these aren't file paths, they're URL paths

path.Join doesn't do well when the protocol/hostname is in play

You end up with `https:///myserver/a.zip` if Join is used.

Reply:
perhaps rename paths to urls?

Reply:
perhaps rename paths to urls?

Reply:
done